kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -128,6 +129,33 @@ def uninstall(
                 f"Release with name {release_name} not found. Could not uninstall app."
             )
 
+    def template(
+        self,
+        release_name: str,
+        chart: str,
+        values: dict,
+        flags: HelmTemplateFlags = HelmTemplateFlags(),
+    ) -> str:
+        """
+        Render chart templates locally and display the output.
+        """
+        with tempfile.NamedTemporaryFile("w") as values_file:
+            yaml.safe_dump(values, values_file)
+            command = [
+                "helm",
+            ]
+            command.extend(
+                [
+                    "template",
+                    release_name,
+                    chart,
+                    "--values",
+                    values_file.name,
+                ]
+            )
```

Comment:
you could merge these

Reply:
I did it like that because I noticed `upgrade_install` is written like that. `Uninstall` isn't though. 

Anyhow, I merged the two.